src/main/java/browserview/BrowserComponent.java

```diff
@@ -512,9 +512,31 @@ public String getCurrentUrl() {
         return driver.getCurrentUrl();
     }
 
+    /**
+     * Switch to the tabs in GitHub PR page
+     * @param tabName Either GithubPageElements.DISCUSSION_TAB, GithubPageElements.COMMITS_TAB
+     *                or GithubPageElements.FILES_TAB
+     */
     public void switchToTab(String tabName) {
         if (GitHubURL.isPullRequestLoaded(getCurrentUrl())) {
-            driver.findElement(By.xpath("//a[@data-container-id='" + tabName + "_bucket']")).click();
+            int tabIndex = 0;
+
+            switch(tabName) {
+                case GithubPageElements.DISCUSSION_TAB:
+                    tabIndex = 1;
+                    break;
+                case GithubPageElements.COMMITS_TAB:
+                    tabIndex = 2;
+                    break;
+                case GithubPageElements.FILES_TAB:
+                    tabIndex = 3;
+                    break;
+                default:
+                    return;
+            }
+
+            String xpath = "//*[@id=\"js-repo-pjax-container\"]/div[2]/div[1]/div/div[2]/div[2]/nav/a[%d]";
+            driver.findElement(By.xpath(String.format(xpath, tabIndex))).click();
```

Comment:
This line can be abstracted to a `click(xpath)` method somewhere (e.g. in BrowserView object)?